frontend/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,74 @@
+import './Footer.css';
+import title from "../../imgs/footerLogo.png";
+import facebook from "../../imgs/facebook.png";
+import twitter from "../../imgs/twitter.png";
+import youtube from "../../imgs/youtube.png";
+import instagram from "../../imgs/ig.png";
+
+function Footer() {
+  return (
+    <div className="main-footer">
+        <div className="row">
+            <div className="column"> 
+                <div className="contact-num">
+                    <h1 className="Contact">Contact: </h1>
+                    <h2 className="number"> (310) 7887-0970 </h2>
+                </div>
+            </div>
+            <div className="column"> 
+                <p className="middle-info"> &copy; 2019 Happy Hats for Kids in Hospitals</p>
+                <p className="middle-info"> Happy hats For Kids In Hospitals is a California 501(C)(3) Nonprofit Organization</p>
+                <p className="middle-info"> Founded in 1991 by Sheri Schrier</p>
```

Comment:
What's the middle-info class?

Reply:
That is just for formatting the middle column text.